### DISCUSSION TITLE
Create a script in powershell

### DISCUSSION THREAD

**Discussion by PrincePIB** [community/community#52295] [Programming Help]
Your department manager comes to you and says that the support team needs tools to perform their tasks. It tells you that you need to validate their needs and find a solution to their request.
A. Needs:

1. a Menu suggesting the following options (Part 1):

1.1. Management of Active Directory objects: management of users, groups and organizational units (Part 2)
•	Create a group
• Create a user
• Add a user to a group
• Display user information
• View group information
• Delete a user
• Delete a group
• Move an object
1.2. Obtaining relevant information from the computer (CIM) (Part 3)
• Query RAM information
• Query processor information
• Request information from hard disk(s)
• Query BIOS information
• Query network card information
• Request for information on the products (software) installed
• Request for information related to the operating system
• Query system information
1.3. Handling with CIM (Part 4)
• Restart a local or remote computer
• View processes and end a process or set of processes
• Validate the state of services before starting/stopping a Windows service

B. After each operation, the menu must offer the possibility of carrying out an additional task (Part 1)
C. To build your script you must use a template which will include, among other things, explanations in comments on your commands. (Part 1)
D. Your script should display an error message in case of problems (Part 1)

**Comment by tuves** [comment]
Hi @PrincePIB,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting screenshots of test questions without context will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
